docs/learn/governance/governance-tech.md

```diff
@@ -1,36 +1,37 @@
 ---
-title: Governance Voting in Shimmer Explained
-description: A technical description of the Governance functionality in Shimmer
+title: Governance Voting Explained
+description: A technical description of the Governance functionality in IOTA & Shimmer
 image: /img/learn/governance/vote-graphic04.png
 keywords:
   - Documentation
   - Firefly
   - Hornet INX
   - Voting
   - Shimmer
+  - IOTA
   - Governance
 ---
 
-On-chain governance is essential for any decentralized DLT. The ability to make decisions through voting functionalities executed on the Blockchain / DAG removes a centralization point and gives token holders the power to make their own decisions. With Shimmer's Firefly Wallet, Shimmer token holders can now participate in the governance of the network.
-With the support of the IOTA Foundation, the Shimmer community has developed a highly flexible and secure system for casting votes directly on the Shimmer Tangle
-through the Firefly Wallet and counting the votes in the network’s Hornet nodes.
+On-chain governance is essential for any decentralized DLT. The ability to make decisions through voting functionalities executed on the Blockchain / DAG removes a centralization point and gives token holders the power to make their own decisions. IOTA and Shimmer token holders can now carry out governance decisions with the Firefly Wallet.
```

Comment:
"Carry out" in this context seems to imply that they execute the voted action instead of voting.

Reply:
Ah yes - that may be confusing, changed that